### DISCUSSION TITLE
So, What is Next? สำหรับเวอร์ชั่น 2.00

### DISCUSSION THREAD

**Discussion by kon3ko** [kon3ko/cover-next#7] [General]
หลังจากได้อัปเดต 2.00 ได้มีการเปลี่ยนแปลงที่สำคัญเข้ามาคือ ในหน้าเลือกไฟล์ของสมาชิกที่ไม่ได้เป็น Premium ของเว็บบิทจะไม่แสดงรูปหน้าปกอีกต่อไป เหตุผลหลักคือ ทางเว็บบิทได้เอาไอค่อนรูป 📷 กล้อง ออกไปจากรายการเลือกไฟล์

![image](https://github.com/kon3ko/cover-next/assets/50349884/8b4a915f-33bf-4e11-b818-a97dd44b13fa)

เป็นเหตุผลที่ผมไม่สามารถเอาลิงก์รูปภาพมาทำภาพหน้าปกได้ 

ดังนั้นทางเลือกมีดังต่อไปนี้

1. เติมเงินเว็บบิทเพื่อเป็นสมาชิก Premium เพื่อที่จะได้มีไอค่อนรูปกล้องกลับมาและใช้งานปลั๊กอินได้เหมือนเดิม
2. ทำเซิร์ฟเวอร์กลางเพื่อ Cache ลิงก์รูปภาพ จะทำให้สมาชิกธรรมดาก็จะมีรูปภาพด้วย (มีค่าใช้จ่ายเซิร์ฟเวอร์เพิ่มเข้ามา ใครจ่าย?)
3. ช่วยกันเรียกร้องให้ทางเว็บ เอาไอค่อนรูปกล้องกลับมา สำหรับสมาชิกธรรมดา
4. ไม่สนใจ ใช้งานต่อไป แบบไม่มีรูปหน้าปก

หรือใครมีข้อเสนอแนะ สามารถเสนอมาได้เลยครับ

**Comment by Nutty789** [comment]
เขียนให้ไปดึงรูปจาก screenshot ไม่ได้หรอครับ

**Reply by kon3ko** [reply]
ถ้าทำแบบนั้น บอทจะวิ่งเข้าไปดึงทุกรายการ ทำให้เว็บล่มได้นะครับ ถ้ามีคนใช้ปลั๊กอินหลายคน ต้องมีเซิร์ฟเวอร์ Cache ครับถึงทำได้

**Comment by T5ive** [comment]
![ภาพ](https://github.com/kon3ko/cover-next/assets/41800233/363adb87-a422-492d-9d13-7cf1d93d40c5)
จากเท่าที่ดูเหมือนยังอยู่และดึงมาได้นะครับ

อาจจะเป็นเพราะเว็บทำการอัพเดทบางส่วนทำให้เกิดบัคในช่วงเวลานั้นมั้ยครับ 
หรือมีการประกาศจากทางแอดมินว่าจะเอาออกหรอครับ

ปล. ผมไม่ได้สมัคร Premium ครับ

**Reply by kon3ko** [reply]
เพิ่งเอากลับมาครับ ตอนนี้ให้ไปใช้ เวอร์ชั่น 2.02 จะกลับมาปกติทุกอย่างแล้วครับ

**Comment by FiRSTTX** [comment]
![image](https://github.com/kon3ko/cover-next/assets/144312867/9222e9e0-dacc-4c31-bd71-bf90991fcd81)

รูปปกไม่ขึ้นอีกแล้วแฮะ BUG หรือต้องสมัคร Premium จริงๆครับ

**Reply by kon3ko** [reply]
ไม่ใช่ Bug ครับ ดูเหมือนว่าเว็บตั้งใจเอาไอค่อนรูปกล้องออกครับ

**Reply by FiRSTTX** [reply]
รับทราบครับ ขอบคุณครับ

**Comment by kon3ko** [comment]
ถ้าเป็นทางออกที่ดีที่สุดคือทำ Server มา Cache แล้วระบบจะใช้งานได้ตามปกติไม่ว่าจะเป็นสมาชิกระดับอะไรก็ตาม
อาจจะมีดีเลย์นิ๊ดหน่อย แต่ประเด็นคือใครจะจ่ายค่าเซิร์ฟเวอร์ครับ หรือใครมีเซิร์ฟเวอร์ฟรีไหมครับ ถ้าแบบนั้น หลังไมค์ผมมาได้เลย

**Reply by poon74761** [reply]
ถ้าทำให้ใช้ได้เฉพาะคนโดเนท เพื่อเอาไปโคฟเวอค่าเซิฟ พอจะเป็นไปได้ไหมครับ

**Reply by kon3ko** [reply]
> ถ้าทำให้ใช้ได้เฉพาะคนโดเนท เพื่อเอาไปโคฟเวอค่าเซิฟ พอจะเป็นไปได้ไหมครับ

ค่าโดเนทครั้งเดียว แต่เซิร์ฟเวอร์รายเดือนครับ ต้องหาคนโดเนทค่าเซิร์ฟเวอร์ครับ 

ผมไปเช็คมาแล้วเดือนละประมาณ 80 บาทปีละ 1,000 ครับ ใครสนใจแบกตรงนี้ทักมานะครับ จ่ายเป็นปีแล้วผมจะทำเซิร์ฟเวอร์ Cache ให้ ทีนี้ต่อให้ไม่มีไอค่อนรูปกล้องก็ยังมีรูปให้ดูอยู่ดีครับ

ส่วนตอนนี้คนที่เติม Premium ไว้ ก็จะยังใช้งานได้ตามปกติครับ

**Reply by kon3ko** [reply]
> ปีละ800ต่อ1คนใช่ไหมครับ

จ่าย 1,000 บาทแค่ครั้งเดียว จะได้ 1 ปี และทุกคนที่ใช้ปลั๊กอินจะใช้งานได้เหมือนกันหมดครับ

คือผมจะเช่าเซิฟเวอร์มา แล้วแก้ไขให้ปลั๊กอินที่ทุกคนโหลดไป ให้ไปใช้งานเซิร์ฟเวอร์ที่ทำขึ้นมาครับ ดังนั้นได้ทุกคนที่ใช้ปลั๊กอินครับ

**Reply by Nutty789** [reply]
เห็นด้วยครับ ปีละ 100 ไม่แพง ใครไม่จ่ายก็ไม่ปลดให้ เหมือน donate แล้ว full function ไปเลย ใช้ได้ทุกคนไม่จำเป็น เพราะถ้าเราเลี้ยง RD เหลือๆ ไม่ต้องเสีย vip ฝั่งนู้น ทางนี้ประหยัดกว่า

**Reply by poon74761** [reply]
เห็นด้วยครับๆ เก็บคนละ100บาท ต่อปี เเล้วก็ให้ใช้เฉพาะคนที่โดเนท

**Comment by eak012** [comment]
เป็น plugin ที่มีประโยชน์และชอบมากๆเลย ใช้ทุกวัน แต่เหตุการณ์แบบนี้ไม่รู้จะข่วยยังไง เอาใจข่วยครับผม

**Comment by johnclassic** [comment]


**Reply by kon3ko** [reply]
ตอนนี้จะดึงมาแค่ส่วนของ description อยากให้ตอนจิ้ม view แล้วดึง ss มาแสดงด้วยหรอครับ

**Reply by kon3ko** [reply]
มันต้องรออะครับแบบนั้น เอาเมาส์วาง รอโหลด พอเราขยับไปอันอื่น ของเก่ายังโหลดไม่เสร็จ ต้องโหลดของใหม่อีกแล้ว ไม่พอยังต้องรอหลายวินาทีกว่าจะแสดงด้วยนะครับ 

ลองนึกภาพ เอาเมาส์วางแล้วต้องรอ 5-10 วินาทีดูครับ ต่อหนึ่งภาพ ถ้าเราต้องดู 10 ภาพ เราจะเสียเวลาดูราว ๆ 1 นาทีครับ แบบนี้สู้คลิกเปิดแทบใหม่ แล้วดูแต่ละอันยังไวกว่าเลยครับ

ผมสรุปให้นะครับ ทำได้แต่แค่ไม่เหมาะสมครับ

**Comment by RoastedCat777** [comment]
ว่าไงว่าตามกันครับ เห็นด้วยทุกอย่าง เตรียมเงินไว้แล้ว 100 บาท

**Comment by BankzazaSeeD** [comment]
ปีล่ะ100 ครับ

**Reply by BankzazaSeeD** [reply]
สมาชิกท่านใด เห็นด้วยกับการเสียค่าบริการpluginทีมีประโยชน์นี้ ต่อคนปีล่ะ100บาท(ผมมองว่าไม่มากไม่มาย)เพื่อเช่าเซิฟเวอร์Cache ลิงก์รูปภาพ เพื่อจะแสดงหน้าปกเหมือนเดิม (สามารถกดโหวตได้ที่เม้นนี้เลยครับ)

**Reply by Nutty789** [reply]
ได้แน่ๆครบ 10 คน เปิดก่อนเลย ไม่เห็นแล้วหงุดหงิด มันชินตา 5555

**Comment by aotzaok** [comment]
เสนออีกทางเลือก คือค่าสมัครแสดงปกคนละ 100 บาท ต่อปี เป็นการไม่บังคับดีครับ

**Comment by kon3ko** [comment]
ผมได้ลองเปิดใช้ Server Cache แล้วนะครับ ลองใช้กันดูนะครับ ทดลองเช่ามา 1 เดือนครับ

**Reply by Nutty789** [reply]
edited update ก่อนแล้วขึ้นปรกติ มีโหลดช้าพอสมควร

**Reply by aotzaok** [reply]
เจ๋งเลยครับ พร้อมซัพพอร์ตครับ

**Reply by kon3ko** [reply]
ทดลองดูก่อนนะครับว่าใช้งานได้ดีไหม

จริง ๆ ระบบนี้ต้องการคนใช้ปลั๊กอินเยอะ ๆ ถึงจะทำงานได้ดีครับ ถ้าคนใช้น้อย ๆ จะทำให้รอโหลดนานครับ (ฝากชวนเพื่อนมาใช้หน่อยนะครับ)

พอมัน Cache ได้ มันก็แทบไม่ต้องรอโหลดเลยครับ

**Reply by BankzazaSeeD** [reply]
เปิดใช้ยังไงครับ55

**Reply by BankzazaSeeD** [reply]
ได้ล่ะครับ ต้องอัพเดทก่อน

**Reply by FlacZa** [reply]
อัพเดทแล้วยังไม่มีรูปเหมือนเดิมเลยครับ

**Reply by Lez9n** [reply]
แจ่มครับ ภาพกลับมาแล้วว

**Reply by rOMANEE1** [reply]
ขอบคุณสังคมแห่งการแบ่งปัน

**Comment by Nutty789** [comment]
ขอช่องทาง donate หน่อยครับ ผมมือไหม่ github หาไม่เจอข้อมูล

**Reply by Lez9n** [reply]
เห็นแปะลิ้ง fb ไว้อยู่ลองติดต่อไปครับ 
https://m.me/100001345584902

**Comment by shadows245** [comment]
มีช่องทาง donate อื่นไหมครับ ผมไม่ได้ใช้ messenger

**Comment by RoastedCat777** [comment]
เว็บเปลี่ยนลิงค์ใหม่ครับภาพไม่แสดง ไม่โชว์รูปเฟือง ปลั๊กอินเลย

**Reply by kon3ko** [reply]
แจ้งปัญหาเปิด Issue ใหม่ได้เลยครับ

**Reply by kon3ko** [reply]
https://github.com/kon3ko/cover-next/issues

**Comment by RoastedCat777** [comment]
เว็บเปลี่ยนลิงค์ใหม่ครับภาพไม่แสดง ไม่โชว์รูปเฟือง ปลั๊กอินเลย
https://bearbit.org/viewno18sbx.php?cat=92
